_data/schedule.yaml

```diff
@@ -0,0 +1,36 @@
+- description: "WildFly 30 Beta: PR submission deadline"
+  date: 2023-09-20
+- description: "WildFly 30 Beta: WildFly Core ready to tag"
+  date: 2023-09-22
+- description: "WildFly 30 Beta: Final merges and checks"
+  date: 2023-09-25
+- description: "WildFly 30 Beta: Buffer day"
+  date: 2023-09-26
```

Comment:
I don't feel strongly about this, but this one maybe doesn't belong on a calendar. There isn't anything specific anyone should do on this day; it's just the gap between "final merges and checks" and "tag and deploy".

In my release schedule announcement I mentioned it mostly just to emphasize that Monday is the deadline and Tuesday is reserved.

Reply:
That's fair, I just copied and pasted from your wildfly-dev email. :) It does seem like a bit of noise to have on a calendar.